README.md

```diff
@@ -133,3 +133,9 @@ To deploy to an environment, run the corresponding command. For example:
 ~~~~
 npm run deploy-development
```

Comment:
I believe `npm run deploy-[env]` will no longer work locally unless you have the expected AWS creds in the local environment (e.g. `AWS_ACCESS_KEY_ID_DEVELOPMENT`). We should either say as much in the README or...

[Daniel and I have been working around this by calling](https://github.com/NYPL/nypl-hold-request-consumer/blob/0ee689cf0fe6ab679bb90b3fc220f448173e764a/.travis.yml#L9-L20) a `travis-deploy` bash script that allows us to leave the existing `deploy-[environment]` commands as-is (i.e. authenticated using `--profile`). The script dynamically replaces the `--profile` param with the inline credentials that travis will have. That way we can continue to deploy locally if needed, but Travis can use those same commands to deploy in the way it needs to.

Reply:
That sounds a good idea! Is there an example of this `travis-deploy`? I can update it.

Reply:
Yep https://github.com/NYPL/nypl-hold-request-consumer/blob/0ee689cf0fe6ab679bb90b3fc220f448173e764a/scripts/travis-deploy.js

If you use it, make sure to make it executable before committing: `chmod +x ./scripts/travis-deploy.js`

Reply:
Thanks!!